examples/a2c/a2c_atari.py

```diff
@@ -0,0 +1,212 @@
+"""
+This script reproduces the Advantage Actor Critic (A2C) Algorithm
+results from Schulman et al. 2017 for the on Atari Environments.
+"""
+
+import hydra
+
+
+@hydra.main(config_path=".", config_name="config_atari", version_base="1.1")
+def main(cfg: "DictConfig"):  # noqa: F821
+
+    import time
+
+    import numpy as np
+    import torch.optim
+    import tqdm
+
+    from tensordict import TensorDict
+    from torchrl.collectors import SyncDataCollector
+    from torchrl.data import LazyMemmapStorage, TensorDictReplayBuffer
+    from torchrl.data.replay_buffers.samplers import SamplerWithoutReplacement
+    from torchrl.envs import ExplorationType, set_exploration_type
+    from torchrl.objectives import A2CLoss
+    from torchrl.objectives.value.advantages import GAE
+    from torchrl.record.loggers import generate_exp_name, get_logger
+    from utils_atari import make_parallel_env, make_ppo_models
+
+    device = "cpu" if not torch.cuda.is_available() else "cuda"
```

Comment:
```suggestion
    device = "cpu" if not torch.cuda.device_count() else "cuda"
```
is_available will be deprecated I think